apps/web/src/app/[locale]/tracks/_components/track-challenge-card.tsx

```diff
@@ -27,8 +30,17 @@ const BGS_BY_DIFFICULTY = {
   EXTREME: 'to-difficulty-extreme/20 dark:to-difficulty-extreme-dark/20',
 } as const;
 
-export function TrackChallenge({ challenge, className, mock }: TrackChallengeProps) {
+const MockComponent = ({ challenge }: MockComponentProps) => (
```

Comment:
`MockComponent` shouldn't return this code unless `mock === true`, so we should also add a new `mock` prop to `MockComponentProps`

Reply:
Sorry @QuiiBz I'm kind of a dumbass , can you tell me why is the mock prop required at the first place. Why can't it be always true ? Are we planning to replace the mock with something in future ?

Reply:
In `feature-card.tsx` we pass a `mock` prop to be true, but in `track-card.tsx` this `mock` prop isn't used and so is false. We should keep this behavior to hide or show the checkbox

Reply:
👍🏻